### ISSUE TITLE
Simple feature request - AWS temporary security credentials

### ISSUE THREAD

**Issue by pierreguilmin** [JMSwag/PyUpdater-S3-Plugin#5]
Hi!

Currently this tool can't be used with **AWS temporary security credentials**. 

What about adding an optional ``self.session_token`` variable parsed from an environment variable like `` PYU_AWS_SESSION_TOKEN`` to allow this feature? It would be really easy to do (~2 lines), I can code it and PR if you want.

In ``s3_uploader.py``:
```diff
class S3Uploader(BaseUploader):

    # ...

    def init_config(self, config):
         # ...
+        self.session_token = os.environ.get(u'PYU_AWS_SESSION_TOKEN')
         # ...

    def _connect(self):
        session = Session(aws_access_key_id=self.access_key,
                          aws_secret_access_key=self.secret_key,
+                         aws_session_token=self.session_token,
                          region_name='us-west-2')

        # ...
```
That's it!

Thank you for the work you did on this project btw.

See also: [class boto3.session.Session documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html#boto3.session.Session)

**Comment by JMSwag** [comment]
Sounds great. I'll get your PR merged

**Comment by JMSwag** [comment]
Closed in #6
